### PR TITLE
Fix the issue `The component Styled(styled.span) has been created dynamically

### PR DESCRIPTION
**Pull request by panichevoleg**
### Prerequisites

Please make sure you can check the following two boxes:

- [x] I have read the **CONTRIBUTING** document
- [x] My code follows the code style of this project

### Contribution Type

What types of changes does your code introduce? Put an `x` in all the boxes that apply:

- [x] Bug fix (non-breaking change which fixes an issue, please reference the issue id)
- [ ] New feature (non-breaking change which adds functionality, make sure to open an associated issue first)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] My change requires a change to the documentation
- [ ] I have updated the documentation accordingly
- [ ] I have added tests to cover my changes
- [ ] All new and existing tests passed

### Description

`Skeleton` component produces the error:

`The component Styled(styled.span) with the id of "sc-fUOEQR" has been created dynamically.
You may see this warning because you've called styled inside another component.
To resolve this only create new StyledComponents outside of any render method and function component.`

The solution is to move creation of `StyledSpan` component out of `render()` method.

### CODE REVIEW

File: src/components/Skeleton/index.tsx [hunk 1]
```diff
@@ -61,30 +61,27 @@ const Span = styled.span`
 
 const randomInt = (min: number, max: number) => Math.floor(Math.random() * (max - min + 1)) + min;
 
+const formatValue = (value: string | number) => {
+  return typeof value === 'number' ? `${value}px` : value;
+};
+
+const StyledSpan = styled(Span)<SkeletonProps>`
+  ${({ count = 1, duration = 1.2, width = '100%', height = '100%', isCircle, isText, isPulsing = true }) => css`
```
Comment (klaidigorishti) [inline src/components/Skeleton/index.tsx:69]: Do you need to add a default `count` here as well? There is already a default coming from `props` in line #81.

File: src/components/Skeleton/index.tsx [hunk 2]
```diff
@@ -61,30 +61,27 @@ const Span = styled.span`
...
+    animation: ${shine} ${duration}s infinite linear ${isPulsing ? 'running' : 'running'};
```
Comment (klaidigorishti) [inline src/components/Skeleton/index.tsx:70]: `isPulsing ? 'running' : 'paused'`

File: src/components/Skeleton/index.tsx (hunk 1 above)
Comment (panichevoleg) [reply src/components/Skeleton/index.tsx:69]: It's not coming from there, I use it there just locally, as max value of loop. But ok, I can pass that value to `StyledSpan` and overwrite value from props.

File: src/components/Skeleton/index.tsx (hunk 2 above)
Comment (panichevoleg) [reply src/components/Skeleton/index.tsx:70]: Right :)

**Review by klaidigorishti — APPROVED** [review]
LGTM! Thanks

**Pull request merged**